docs/_snippets/patient-address.mdx

```diff
@@ -20,7 +20,7 @@
     </ParamField>
 
     <ParamField body="country" type="string">
-      Must be "USA".
+      If specified must be "USA", otherwise will default to "USA"
```

Comment:
nit: punctuation - `will default to "USA".`

Reply:
nit: punctuation - `If specified, must be "USA"`